src/kolmogorov_solve.jl

```diff
@@ -17,21 +17,29 @@ function DiffEqBase.solve(
     )
 
     tspan = prob.tspan
-    phi = prob.phi
-    xspan = prob.xspan
-    sigma = prob.sigma
-    μ = prob.μ
-    d = prob.d
+    sigma = prob.g
+    μ = prob.f
+    if prob isa SDEProblem
+        xspan = prob.kwargs.data.xspan
+        d = prob.kwargs.data.d
+        u0 = prob.u0
+        phi(xi) = pdf.(u0 , xi)
```

Comment:
this is always going to assume independence. We probably want to handle multivariate distributions as well.

Reply:
Yes I realized this just now when I was conductiing some high dimensional tests. Changing this in the next PR